internal/security/config/main.go

```diff
@@ -51,19 +51,21 @@ func Main(ctx context.Context, cancel context.CancelFunc) int {
 
 	serviceHandler := NewBootstrap()
 
-	bootstrap.Run(
+	_, _, _ = bootstrap.RunAndReturnWaitGroup(
```

Comment:
```suggestion
	bootstrap.RunAndReturnWaitGroup(
```

Reply:
@farshidtz those empty returns to avoid Golang linting error

Reply:
What thinking the linter would complain, but that is only if it is an error message. Removed.

Reply:
okay, good to know thanks.